### ISSUE TITLE
Search result page should not have an h4

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1615]
## Problem
The form with additional sources below the search result with the text 'Not finding what you're looking for...' should not be a heading. This is located in `VMetaSearchForm.vue` component:

<img width="708" alt="Not finding what you're looking for screenshot" src="https://user-images.githubusercontent.com/15233243/183044595-abeb7f37-d017-4778-baf8-cda9b469591f.png">

## Description
On the search result page (http://search.openverse.engineering/?q=cat), it is currently an h4. The visual styles should match the Figma mockups: [`md` and up](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1370%3A19752) and [mobile](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1393%3A20071).
Please, note that the current implementation still uses buttons and not a `select` for sources as in the mockup. That change will be implemented later.

**Comment by RARNE12** [comment]
Hi @obulat, just to clarify, do you want us to change h4 to h1 with the appropriate Tailwinds CSS styling class?
cc, @kickeyhickey

**Comment by obulat** [comment]
@RARNE12, on this page, the heading should be replaced with a paragraph (`<p>`), with the appropriate Tailwind styling classes (both for mobile and for larger screens).

**Comment by RARNE12** [comment]
@obulat, thank you for your quick response and suggestions. @kickeyhickey and I will start working on it at our earliest. Have a great day.

**Comment by Jbudd512** [comment]
@obulat Hello, I'm working with @RARNE12 and @kickeyhickey on this ticket. We worked on this yesterday and I'm pushing it up for review. Please let us know if these changes are good.

**Comment by Jbudd512** [comment]
Pull request link: https://github.com/WordPress/openverse-frontend/pull/1647

**Comment by obulat** [comment]
@Jbudd512, Thank you for your PR! If you add `Fixes: #<issue number>` to your PR's description, it links it to this issue automatically :) I will be reviewing your PR soon.
